### ISSUE TITLE
creating account with invalid namespacestore should throw proper message

### ISSUE THREAD

**Issue by bssrikanth** [noobaa/noobaa-operator#650]
### Environment info
- NooBaa Version: **VERSION**
INFO[0000] CLI version: 5.9.0                           
INFO[0000] noobaa-image: noobaa/noobaa-core:master-20210607 
INFO[0000] operator-image: noobaa/noobaa-operator:5.9.0 

- Platform: **Kubernetes 1.14.1 | minikube 1.1.1 | OpenShift 4.1 | other: specify**
Server Version: 4.7.12
Kubernetes Version: v1.20.0+df9c838

### Actual behavior

Currently when users are created with invalid default_resource [i.e  namespacestore] noobaa response throws below error

```
ERRO[0001] ⚠️  RPC: account.create_account() Response Error: Code=INTERNAL Message=Cannot read property 'nsfs_config' of undefined 
FATA[0001] ❌ Cannot read property 'nsfs_config' of undefined 

```
This error should be improved to mention that user did not pass a valid namespacestore default_resource.

### Expected behavior
creating account with non-existent default_resource should throw  NO_SUCH_RESOURCE or similar message

### Steps to reproduce
Try to create noobaa account with non-existent default_resource:
```
[root@hpoocpci-inf HPO]# noobaa namespacestore list
NAME   TYPE   TARGET-BUCKET   PHASE   AGE        
fs1    nsfs                   Ready   4h27m42s   

Run account create :

[root@hpoocpci-inf HPO]# /usr/local/bin/noobaa api account_api create_account '{"email": "user01@mail.com","name": "user01","has_login": false,"s3_access": true,"allowed_buckets": { "full_permission": true },"default_resource": "fds1","nsfs_account_config": { "uid": 222,"gid": 223,"new_buckets_path": "user01_buckets" }}'
```

**Comment by bssrikanth** [comment]
I dont see the nsfs tag after transfer of this issue from noobaa-core.

**Comment by bssrikanth** [comment]
This issue seems to be fixed with 0622 build. Can you please link this to the PR which fixed it?

```
#  /usr/local/bin/noobaa api account_api create_account '{"email": "user01@mail.com","name": "user01","has_login": false,"s3_access": true,"allowed_buckets": { "full_permission": true },"default_resource": "fds1","nsfs_account_config": { "uid": 222,"gid": 223,"new_buckets_path": "user01_buckets" }}'
INFO[0001] ✅ Exists: NooBaa "noobaa"                    
INFO[0001] ✅ Exists: Service "noobaa-mgmt"              
INFO[0001] ✅ Exists: Secret "noobaa-operator"           
INFO[0001] ✅ Exists: Secret "noobaa-admin"              
INFO[0001] ✈️  RPC: account.create_account() Request: map[allowed_buckets:map[full_permission:true] default_resource:fds1 email:user01@mail.com has_login:false name:user01 nsfs_account_config:map[gid:223 new_buckets_path:user01_buckets uid:222] s3_access:true] 
WARN[0001] RPC: GetConnection creating connection to wss://localhost:36339/rpc/ 0xc00078d130 
INFO[0001] RPC: Connecting websocket (0xc00078d130) &{RPC:0xc00003ad70 Address:wss://localhost:36339/rpc/ State:init WS:<nil> PendingRequests:map[] NextRequestID:0 Lock:{state:1 sema:0} ReconnectDelay:0s} 
INFO[0001] RPC: Connected websocket (0xc00078d130) &{RPC:0xc00003ad70 Address:wss://localhost:36339/rpc/ State:init WS:<nil> PendingRequests:map[] NextRequestID:0 Lock:{state:1 sema:0} ReconnectDelay:0s} 
ERRO[0001] ⚠️  RPC: account.create_account() Response Error: Code=BAD_REQUEST Message=default resource doesn't exist 
FATA[0001] ❌ default resource doesn't exist             

# noobaa version
INFO[0000] CLI version: 5.9.0                           
INFO[0000] noobaa-image: noobaa/noobaa-core:master-20210622 
INFO[0000] operator-image: noobaa/noobaa-operator:5.9.0 

```

**Comment by nimrod-becker** [comment]
I'm not aware of actively fixing this, but we might have fixed something larger which also fixed this
@romayalon any ideas?

**Comment by romayalon** [comment]
yes this issue is a duplicate of https://github.com/noobaa/noobaa-core/issues/6586, fixed it in https://github.com/noobaa/noobaa-core/pull/6590

**Comment by nimrod-becker** [comment]
Fixed with noobaa/noobaa-core#6590
